lib/spreadsheet/worksheet.rb

```diff
@@ -232,6 +232,9 @@ def insert_row idx, cells=[]
       updated_from idx
       res
     end
+    def << cells=[]
+      insert_row last_row_index + 1, cells
```

Comment:
it fails when adding the first row, because `last_row_index` returns 0 when sheet is empty.

Reply:
```ruby
insert_row rows.size, cells
```